tests/nn/checkpoint/test_checkpoint_activations.py

```diff
@@ -267,3 +267,38 @@ def test_deprecated_path():
     # Check if direct import works as before.
     ffn = nn.Sequential(nn.Linear(32, 128), nn.Dropout(p=0.5), nn.Linear(128, 32),)
     ffn = deprecated_checkpoint_wrapper(ffn, {})
+
+
+@skip_if_no_cuda
+def test_list_input():
+    """Test to ensure that we correctly does the checkpoint when the input is a list.
```

Comment:
nit: rephrase? perhaps replace "correctly does the checkpoint" with "Test checkpointing with input type lists" or something along those lines.

Reply:
Sure